src/components/Points/PointImage.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { CardMedia } from "@mui/material";
+
+function PointImage({ data }) {
+  const renderImages = () => {
+    const images = [];
+
+    // Loop through the object's properties
+    for (const key in data) {
+      if (typeof data[key] == "string") {
+        // Check if the value is an image URL
+        const isImage =
+          data[key].indexOf(".jpg") > -1 ||
+          data[key].indexOf(".png") > -1 ||
+          (data[key].indexOf(".jpeg") > -1 &&
+            (data[key].indexOf("http") > -1 ||
+              data[key].indexOf("https") > -1));
```

Comment:
Could be simplified:

Check for url with:
```
new URL(...)
```

Check for extension with regex:

```
function isImgUrl(url) {
  return /\.(jpg|jpeg|png)$/.test(url)
}
```

Reply:
Additionally, we can avoid problem with url params by checking `.pathname`, not the whole URL